### DISCUSSION TITLE
Prevent rotation of object while rotating face.

### DISCUSSION THREAD

**Discussion by Najaf93** [jeeliz/jeelizFaceFilter#244] [General]
Hi Guys,
 I am trying to create Jewellery Try On and facing the rotation issue. For example, if I augment a necklace using FaceFilter it rotates whenever I rotate my head. How do I restrict the rotation for a specific object? Please help me to figure this out. I am new to three.js and Jeeliz. I appreciate any replies. Thank You.

**Comment by xavierjs** [comment]
Hi,
Duplicate of https://github.com/jeeliz/jeelizFaceFilter/issues/177
Three.js related issue => ask on https://discourse.threejs.org/

**Reply by Najaf93** [reply]
Thank you, I will drop the question there.

**Reply by Najaf93** [reply]
I have seen that #177 , but I don't know where to find the pose matrix. Really sorry for the noob question, i just started working on this.
